_docs-v3/custom-views/visibleRange.md

```diff
@@ -58,7 +58,7 @@ $('#calendar').fullCalendar({
 If you need to change the `visibleRange` after initialization, you can do that using the standard technique for [setting options dynamically](dynamic-options):
 
 ```js
-$('#calendar').fullCalendar('option', 'visibleRange', {
+new Calendar.option('visibleRange', {
```

Comment:
same issue here. it's not syntactically correct, should be `(new Calendar()).option(` instead, but regardless, let's assume a global `calendar` object is globally available.